portfolio/src/main/webapp/baking.html

```diff
@@ -7,17 +7,21 @@
   </head>
   <body>
     <div id="content">
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="about-me.html">About Me</a></li>
+        <li><a href="baking.html" class="active">Food</a></li>
+      </ul>        
       <h1>Food Gallery:</h1>
-      <div class="macaron-thumbnails">
+      <div class="thumbnails">
         <p>Macarons:</p>
         <img src="/images/appo_mac.jpg" height=250 alt="Spiced Apple Macarons">
         <img src="/images/mocha_mac.jpg" height=250 alt="Mocha Macarons">
         <img src="/images/pep_mac.jpg" height=250 alt="Peppermint Chocolate Macarons">
         <img src="/images/raz_mac.jpg" height=250 alt="Raspberry Macarons">
         <img src="/images/rv_mac.jpg" height=250 alt="Red Velvet Macarons">
-
         <p>Not macarons:</p>
-        <img src="/images/biscotti.jpg" height=250 alt="Hazelnut Chocolate Biscotti">
+        <img src="/images/biscotti.jpg" height=250 alt="Hazelnut Chocolate Biscotti">        
```

Comment:
nit: seems to be trailing whitespaces at the end of this line? Please remove them.

https://google.github.io/styleguide/htmlcssguide.html#Trailing_Whitespace